tests/rptest/tests/acls_test.py

```diff
@@ -44,6 +44,9 @@ class AccessControlListTest(RedpandaTest):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, num_brokers=3, **kwargs)
+        self.base_user_cert = None
```

Comment:
these three variables seem to be used (and in the case of base_user_cert) replace the old calculation of `cert` passed to rpk, but i don't see where these variables are set.

```python
        self.base_user_cert = None
        self.cluster_describe_user_cert = None
        self.admin_user_cert = None
```

is that a mistake or they are set somewhere else i'm not seeing?

Reply:
It fixes a lint error and simplifies the code a little, the assignment is here: https://github.com/BenPope/redpanda/blob/3ed6fd8efb5e409f4ca49ed35d2f0125437874af/tests/rptest/tests/acls_test.py#L85-L103